
[     UTC     ] Logs for evandrocleto-portifolio-home-4hkr3m.streamlit.app/
────────────────────────────────────────────────────────────────────────────────────────
[16:10:54] 🖥 Provisioning machine...
[16:10:59] 🎛 Preparing system...
[16:11:05] ⛓ Spinning up manager process...
[16:11:10] 🎈 Inflating balloons...
[16:09:12] 🚀 Starting up repository: 'portifolio', branch: 'main', main module: 'Home.py'
[16:09:12] 🐙 Cloning repository...
[16:09:14] 🐙 Cloning into '/app/portifolio'...

[16:09:14] 🐙 Cloned repository!
[16:09:14] 🐙 Pulling code changes from Github...
[16:09:15] 📦 Processing dependencies...
Check if streamlit is installed

──────────────────────────────── Installing Streamlit ──────────────────────────────────

Collecting streamlit
  Downloading streamlit-1.22.0-py2.py3-none-any.whl (8.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.9/8.9 MB 97.5 MB/s eta 0:00:00[2023-05-22 16:09:16.603867] 
Collecting requests>=2.4
  Downloading requests-2.31.0-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 KB 188.7 MB/s eta 0:00:00[2023-05-22 16:09:16.750509] 
Collecting protobuf<4,>=3.12
  Downloading protobuf-3.20.3-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 148.6 MB/s eta 0:00:00[2023-05-22 16:09:17.250816] 
Collecting tenacity<9,>=8.0.0
  Downloading tenacity-8.2.2-py3-none-any.whl (24 kB)
Collecting cachetools>=4.0
  Downloading cachetools-5.3.0-py3-none-any.whl (9.3 kB)
Collecting tornado>=6.0.3
  Downloading tornado-6.3.2-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (426 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 426.9/426.9 KB 214.4 MB/s eta 0:00:00[2023-05-22 16:09:17.504107] 
Collecting importlib-metadata>=1.4
  Downloading importlib_metadata-6.6.0-py3-none-any.whl (22 kB)
Collecting pydeck>=0.1.dev5
  Downloading pydeck-0.8.1b0-py2.py3-none-any.whl (4.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.8/4.8 MB 195.3 MB/s eta 0:00:00[2023-05-22 16:09:17.695347] 
Collecting rich>=10.11.0
  Downloading rich-13.3.5-py3-none-any.whl (238 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 238.7/238.7 KB 196.5 MB/s eta 0:00:00[2023-05-22 16:09:17.845701] 
Collecting numpy
  Downloading numpy-1.24.3-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 17.3/17.3 MB 218.1 MB/s eta 0:00:00[2023-05-22 16:09:18.420140] 
Collecting packaging>=14.1
  Downloading packaging-23.1-py3-none-any.whl (48 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.9/48.9 KB 173.3 MB/s eta 0:00:00[2023-05-22 16:09:18.507080] 
Collecting python-dateutil
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 KB 216.3 MB/s eta 0:00:00[2023-05-22 16:09:18.553527] 
Collecting pillow>=6.2.0
  Downloading Pillow-9.5.0-cp39-cp39-manylinux_2_28_x86_64.whl (3.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.4/3.4 MB 162.4 MB/s eta 0:00:00[2023-05-22 16:09:18.996883] 
Collecting altair<5,>=3.2.0
  Downloading altair-4.2.2-py3-none-any.whl (813 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 813.6/813.6 KB 156.5 MB/s eta 0:00:00[2023-05-22 16:09:19.098903] 
Collecting pyarrow>=4.0
  Downloading pyarrow-12.0.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (39.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 39.0/39.0 MB 156.6 MB/s eta 0:00:00[2023-05-22 16:09:19.589840] 
Collecting typing-extensions>=3.10.0.0
  Downloading typing_extensions-4.5.0-py3-none-any.whl (27 kB)
Collecting pandas<3,>=0.25
  Downloading pandas-2.0.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.4/12.4 MB 161.4 MB/s eta 0:00:00[2023-05-22 16:09:20.095499] 
Collecting validators>=0.2
  Downloading validators-0.20.0.tar.gz (30 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting blinker>=1.0.0
  Downloading blinker-1.6.2-py3-none-any.whl (13 kB)
Collecting watchdog
  Downloading watchdog-3.0.0-py3-none-manylinux2014_x86_64.whl (82 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 82.1/82.1 KB 193.6 MB/s eta 0:00:00[2023-05-22 16:09:20.707821] 
Collecting pympler>=0.9
  Downloading Pympler-1.0.1-py3-none-any.whl (164 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 164.8/164.8 KB 206.1 MB/s eta 0:00:00[2023-05-22 16:09:20.744639] 
Collecting toml
  Downloading toml-0.10.2-py2.py3-none-any.whl (16 kB)
Collecting click>=7.0
  Downloading click-8.1.3-py3-none-any.whl (96 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 96.6/96.6 KB 229.9 MB/s eta 0:00:00[2023-05-22 16:09:20.834714] 
Collecting gitpython!=3.1.19
  Downloading GitPython-3.1.31-py3-none-any.whl (184 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 184.3/184.3 KB 207.4 MB/s eta 0:00:00[2023-05-22 16:09:20.916177] 
Collecting tzlocal>=1.1
  Downloading tzlocal-5.0.1-py3-none-any.whl (20 kB)
Collecting entrypoints
  Downloading entrypoints-0.4-py3-none-any.whl (5.3 kB)
Collecting toolz
  Downloading toolz-0.12.0-py3-none-any.whl (55 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.8/55.8 KB 141.8 MB/s eta 0:00:00[2023-05-22 16:09:21.061588] 
Collecting jinja2
  Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.1/133.1 KB 127.4 MB/s eta 0:00:00[2023-05-22 16:09:21.119564] 
Collecting jsonschema>=3.0
  Downloading jsonschema-4.17.3-py3-none-any.whl (90 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 90.4/90.4 KB 151.2 MB/s eta 0:00:00[2023-05-22 16:09:21.212108] 
Collecting gitdb<5,>=4.0.1
  Downloading gitdb-4.0.10-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.7/62.7 KB 192.3 MB/s eta 0:00:00[2023-05-22 16:09:21.265110] 
Collecting zipp>=0.5
  Downloading zipp-3.15.0-py3-none-any.whl (6.8 kB)
Collecting pytz>=2020.1
  Downloading pytz-2023.3-py2.py3-none-any.whl (502 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.3/502.3 KB 233.8 MB/s eta 0:00:00[2023-05-22 16:09:21.564933] 
Collecting tzdata>=2022.1
  Downloading tzdata-2023.3-py2.py3-none-any.whl (341 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 341.8/341.8 KB 135.2 MB/s eta 0:00:00[2023-05-22 16:09:21.628479] 
Collecting six>=1.5
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 KB 122.5 MB/s eta 0:00:00[2023-05-22 16:09:21.865454] 
Collecting charset-normalizer<4,>=2
  Downloading charset_normalizer-3.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (199 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 199.2/199.2 KB 188.6 MB/s eta 0:00:00[2023-05-22 16:09:22.105408] 
Collecting certifi>=2017.4.17
  Downloading certifi-2023.5.7-py3-none-any.whl (156 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 157.0/157.0 KB 196.8 MB/s eta 0:00:00[2023-05-22 16:09:22.160937] 
Collecting urllib3<3,>=1.21.1
  Downloading urllib3-2.0.2-py3-none-any.whl (123 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 123.2/123.2 KB 152.6 MB/s eta 0:00:00
Collecting pygments<3.0.0,>=2.13.0
  Downloading Pygments-2.15.1-py3-none-any.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 230.1 MB/s eta 0:00:00[2023-05-22 16:09:22.330354] 
Collecting markdown-it-py<3.0.0,>=2.2.0
  Downloading markdown_it_py-2.2.0-py3-none-any.whl (84 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 84.5/84.5 KB 123.0 MB/s eta 0:00:00[2023-05-22 16:09:22.391783] 
Collecting decorator>=3.4.0
  Downloading decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting smmap<6,>=3.0.1
  Downloading smmap-5.0.0-py3-none-any.whl (24 kB)
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.1.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
Collecting pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0
  Downloading pyrsistent-0.19.3-py3-none-any.whl (57 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.5/57.5 KB 140.6 MB/s eta 0:00:00[2023-05-22 16:09:22.808325] 
Collecting attrs>=17.4.0
  Downloading attrs-23.1.0-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.2/61.2 KB 175.4 MB/s eta 0:00:00[2023-05-22 16:09:22.860135] 
Collecting mdurl~=0.1
  Downloading mdurl-0.1.2-py3-none-any.whl (10.0 kB)
Building wheels for collected packages: validators
  Building wheel for validators (setup.py): started
  Building wheel for validators (setup.py): finished with status 'done'
  Created wheel for validators: filename=validators-0.20.0-py3-none-any.whl size=19579 sha256=77df07c3317a841947fa6414bfa5f5466c4dcb06cedb4150c32ea3ea244f0daa
  Stored in directory: /tmp/pip-ephem-wheel-cache-ajq5moiu/wheels/2d/f0/a8/1094fca7a7e5d0d12ff56e0c64675d72aa5cc81a5fc200e849
Successfully built validators
Installing collected packages: pytz, zipp, watchdog, urllib3, tzlocal, tzdata, typing-extensions, tornado, toolz, toml, tenacity, smmap, six, pyrsistent, pympler, pygments, protobuf, pillow, packaging, numpy, mdurl, MarkupSafe, idna, entrypoints, decorator, click, charset-normalizer, certifi, cachetools, blinker, attrs, validators, requests, python-dateutil, pyarrow, markdown-it-py, jsonschema, jinja2, importlib-metadata, gitdb, rich, pydeck, pandas, gitpython, altair, streamlit
Successfully installed MarkupSafe-2.1.2 altair-4.2.2 attrs-23.1.0 blinker-1.6.2 cachetools-5.3.0 certifi-2023.5.7 charset-normalizer-3.1.0 click-8.1.3 decorator-5.1.1 entrypoints-0.4 gitdb-4.0.10 gitpython-3.1.31 idna-3.4 importlib-metadata-6.6.0 jinja2-3.1.2 jsonschema-4.17.3 markdown-it-py-2.2.0 mdurl-0.1.2 numpy-1.24.3 packaging-23.1 pandas-2.0.1 pillow-9.5.0 protobuf-3.20.3 pyarrow-12.0.0 pydeck-0.8.1b0 pygments-2.15.1 pympler-1.0.1 pyrsistent-0.19.3 python-dateutil-2.8.2 pytz-2023.3 requests-2.31.0 rich-13.3.5 six-1.16.0 smmap-5.0.0 streamlit-1.22.0 tenacity-8.2.2 toml-0.10.2 toolz-0.12.0 tornado-6.3.2 typing-extensions-4.5.0 tzdata-2023.3 tzlocal-5.0.1 urllib3-2.0.2 validators-0.20.0 watchdog-3.0.0 zipp-3.15.0
WARNING: You are using pip version 22.0.3; however, version 23.1.2 is available.
You should consider upgrading via the '/home/appuser/venv/bin/python -m pip install --upgrade pip' command.

────────────────────────────────────────────────────────────────────────────────────────

[16:09:37] 📦 Processed dependencies!

Collecting usage statistics. To deactivate, set browser.gatherUsageStats to False.




2023-05-22 16:10:08.296 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/3_⚛️_Projetos_em_Data_Science.py", line 54, in <module>
    image = Image.open('Projeto02R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'Projeto02R.png'
2023-05-22 16:10:10.220 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/4_🧱_Projetos_Acadêmicos_em_Data_Science.py", line 45, in <module>
    image2 = Image.open('D:\Portifolio\Page_4-Projetos_Academicos\imagens\Ser_Temp_R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\Portifolio\\Page_4-Projetos_Academicos\\imagens\\Ser_Temp_R.png'
2023-05-22 16:10:29.222 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/4_🧱_Projetos_Acadêmicos_em_Data_Science.py", line 45, in <module>
    image2 = Image.open('D:\Portifolio\Page_4-Projetos_Academicos\imagens\Ser_Temp_R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\Portifolio\\Page_4-Projetos_Academicos\\imagens\\Ser_Temp_R.png'
2023-05-22 16:11:46.610 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/3_⚛️_Projetos_em_Data_Science.py", line 54, in <module>
    image = Image.open('Projeto02R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'Projeto02R.png'
2023-05-22 16:12:07.395 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/4_🧱_Projetos_Acadêmicos_em_Data_Science.py", line 45, in <module>
    image2 = Image.open('D:\Portifolio\Page_4-Projetos_Academicos\imagens\Ser_Temp_R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\Portifolio\\Page_4-Projetos_Academicos\\imagens\\Ser_Temp_R.png'
2023-05-22 16:14:17.575 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/3_⚛️_Projetos_em_Data_Science.py", line 54, in <module>
    image = Image.open('Projeto02R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'Projeto02R.png'
2023-05-22 16:18:07.568 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/3_⚛️_Projetos_em_Data_Science.py", line 54, in <module>
    image = Image.open('Projeto02R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'Projeto02R.png'
2023-05-22 16:22:04.573 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
  File "/app/portifolio/pages/3_⚛️_Projetos_em_Data_Science.py", line 54, in <module>
    image = Image.open('Projeto02R.png')
  File "/home/appuser/venv/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'Projeto02R.png'